src/Algebra/Properties/CancellativeCommutativeSemiring.agda

```diff
@@ -0,0 +1,73 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some theory for CancellativeCommutativeSemiring.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra using (CancellativeCommutativeSemiring)
+import Algebra.GCD
+import Algebra.Properties.Semigroup.Divisibility as SemigroupDiv
+import Algebra.Properties.Semiring as SemiringProp
+open import Data.Product using (_×_; _,_; proj₁; proj₂)
+open import Data.Sum using (_⊎_; inj₁; inj₂)
+open import Function using (case_of_; flip)
+open import Level using (_⊔_)
+open import Relation.Binary using (Rel; Symmetric; Setoid)
+open import Relation.Unary using (Pred)
+
+module Algebra.Properties.CancellativeCommutativeSemiring
+  {α ℓ} (R : CancellativeCommutativeSemiring α ℓ)
+  (open CancellativeCommutativeSemiring R)
+  (0≉1 : 0# ≉ 1#)
+  where
+
+open SemigroupDiv *-semigroup using (_∣_; _∤_; ∣-trans)
+open SemiringProp semiring using (∣nonzero⇒≉0)
+open Algebra.GCD _≈_ 0# 1# _*_ using (IsIrreducible; IsPrime; Coprime; GCD)
+
+private A = Carrier
```

Comment:
Do this via the `rename` keyword when opening the bundle.

Reply:
Done.